src/main/java/org/opensearch/geospatial/rest/action/upload/geojson/RestUploadGeoJSONAction.java

```diff
@@ -40,20 +42,53 @@ public String getName() {
      * Supported Routes are
      * POST /_plugins/geospatial/geojson/_upload
      * {
-     *     //TODO https://github.com/opensearch-project/geospatial/issues/29 (implement request body)
+     *   "index": "create_new_index",
+     *   "field" : "geospatial field name",
+     *   "type" : "geospatial field type",
+     *   "data" : [
+     *    {
+     *       "type": "Feature",
+     *       "geometry": {
+     *          "type": "Polygon",
+     *          "coordinates": [
+     *                [
+     *                    [100.0, 0.0],
+     *                    [101.0, 0.0],
+     *                    [101.0, 1.0],
+     *                    [100.0, 1.0],
+     *                    [100.0, 0.0]
+     *                ]
+     *           ]
+     *       },
+     *       "properties": {
+     *          "prop0": "value0",
+     *          "prop1": {
+     *             "this": "that"
+     *          }
+     *      }
+     *   }
+     *  ]
      * }
-     *
+     * PUT /_plugins/geospatial/geojson/_upload
```

Comment:
General question: Why do we need the _upload portion for these requests? Why cant we just use `/_plugins/geospatial/geojson`

Reply:
Ack.

Reply:
Im wondering  if there are other actions here that would warrant a post/put just as much as uploading. Can you think of any? 

I guess geojson isnt a resource. It seems more to me like a function. So maybe _upload does make sense. What are your thoughts @VijayanB ?

Reply:
Let's keep _upload, similar to _bulk where it creates multiple resources based on payload.